### PR TITLE
updates to translation documentation

### PR DESCRIPTION
**Pull request by veenaiyuri**
### Summary

This PR updates the documentation to contribute to Qiskit Documentation translations.

### CODE REVIEW

File: docs/contributing_to_qiskit.rst [hunk 1]
```diff
@@ -558,6 +558,8 @@ repository. To contribute to Qiskit localization, please follow these steps:
         to the PR**.
 #. You will be asked to sign the Qiskit Contributors License Agreement (CLA);
    please do so.
+#. A minimum of three contributors per language are necessary for any new languages to be added to receive official support from the administrators of the localization project.
```
Comment (SooluThomas) [inline docs/contributing_to_qiskit.rst]:
```suggestion
#. A minimum of **three contributors** per language are necessary for any new languages to be added to receive official support from the administrators of the localization project.
```

File: docs/contributing_to_qiskit.rst [hunk 2]
```diff
@@ -558,6 +558,12 @@ repository. To contribute to Qiskit localization, please follow these steps:
         to the PR**.
 #. You will be asked to sign the Qiskit Contributors License Agreement (CLA);
    please do so.
+#. A minimum of **three contributors** per language are necessary for any new
+   languages to be added to receive official support from the administrators of
```
Comment (abbycross) [inline docs/contributing_to_qiskit.rst]:
```suggestion
   languages to be added, to receive official support from the administrators of
```

**Review by abbycross — DISMISSED** [review]
Looks great! I just added a suggestion for one comma, but it's good to go.

**Comment by SooluThomas** [comment]
Thank you Abby for reviewing ❤️

**Review by SooluThomas — DISMISSED** [review]
This PR is good to go!

File: docs/contributing_to_qiskit.rst [hunk 3]
```diff
@@ -558,6 +558,12 @@ repository. To contribute to Qiskit localization, please follow these steps:
         to the PR**.
 #. You will be asked to sign the Qiskit Contributors License Agreement (CLA);
    please do so.
+#. A minimum of **three contributors** per language are necessary for any new
+   languages to be added, to receive official support from the administrators of
+   the localization project.
+#. Among the group of contributors, a translation lead must be identified to serve
+   as a liaison with the administrators of the localization project.
```
Comment (SooluThomas) [inline docs/contributing_to_qiskit.rst]:
```suggestion
   as a liaison with the administrators of the localization project.
```

**Review by SooluThomas — APPROVED** [review]

**Pull request merged**